exp/api/v1beta2/ocimanagedmachinepool_webhook_test.go

```diff
@@ -146,10 +146,12 @@ func TestOCIManagedMachinePool_ValidateCreate(t *testing.T) {
 
 func TestOCIManagedMachinePool_ValidateUpdate(t *testing.T) {
 	validVersion := common.String("v1.25.1")
+	oldVersion := common.String("v1.24.1")
```

Comment:
Should these versions be updated to OKE latest versions?

validVersion: v1.26.2
oldVersion: v1.25.4

Reply:
I think its ok as these are for just unit tests right?